src/pages/SchoolPage.js

```diff
@@ -1,13 +1,15 @@
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import qs from "qs";
-import styled from "styled-components";
+import styled, { keyframes } from "styled-components";
 import { Link } from "react-router-dom";
+import { grey } from "@material-ui/core/colors";
 
 export default function SchoolPage({ location }) {
   const { school } = qs.parse(location.search, {
     ignoreQueryPrefix: true,
   });
+  const { schoolName } = location.state ? location.state : "";
```

Comment:
(Without testing) This doesnt look like it would work if our location.state does not exist. Possibly the else condition should give `{ schoolName: "" }` so there wont be any errors with the destructuring. Also in that case we can just do:

```js
const { schoolName } = location.state ?? { schoolName: school };
```